sbi/src/api/tee_host.rs

```diff
@@ -165,9 +212,9 @@ pub fn get_vcpu_register_set(vmid: u64, index: u64) -> Result<RegisterSetLocatio
 /// # Safety
 ///
 /// The caller must own the pages referenced by `shared_page_addr`, for the number of pages
-/// sufficient to hold the structure layout returned from `get_vcpu_mem_layout()`. Since memory
-/// within the shared-memory state area may be read or written by the TSM at any time, the caller
-/// must treat the memory as volatile for the lifetime of the TVM.
+/// sufficient to hold the `TvmCpuSharedState` strucutre. Since memory within the shared-memory state
```

Comment:
typo: `strucutre` instead of `structure`

Reply:
Done.